source/postprocess/global_statistics.cc

```diff
@@ -0,0 +1,296 @@
+/*
+  Copyright (C) 2017 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/postprocess/global_statistics.h>
+#include <aspect/simulator.h>
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+    template <int dim>
+    void
+    GlobalStatistics<dim>::initialize()
+    {
+      this->get_signals().post_stokes_solver.connect(std_cxx11::bind(&aspect::Postprocess::GlobalStatistics<dim>::store_stokes_solver_history,
+                                                                     std_cxx11::ref(*this),
+                                                                     /*std_cxx11::_1,*/
```

Comment:
This took me a while to understand -- I think what you're saying is that you are ignoring the first argument of the signal (the `SimulatorAccess` reference), right? Instead of this comment, maybe just say `/* Drop first argument of the signal when calling this slot */`